### PR TITLE
SDK demo gets a new blog directory.

### PR DESCRIPTION
**Pull request by wes-turner**
* SDK demo added
* Blog directory added
* Top-level README edited to add new blog section

### CODE REVIEW

**Review by MikhailKardash — APPROVED** [review]
Just one small formatting fix, otherwise LGTM.

**Comment by KevinMusgrave** [comment]
This looks great! I'm testing it now

**Comment by KevinMusgrave** [comment]
It works

File: blog/python_sdk_demo/mednist_model/config.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    metric: val_accuracy
+    smaller_is_better: false
+    max_length: 
+      epochs: 1
+    max_trials: 3
+    mode: aggressive
+resources:
+    slots_per_trial: 1
+entrypoint: model_def:MyMEDMnistTrial
+max_restarts: 0
```
Comment (MikhailKardash) [inline blog/python_sdk_demo/mednist_model/config.yaml]: newline

File: README.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 
 ## Tutorials
 
 | Example                                                       | Dataset          | Framework             |
 |:-------------------------------------------------------------:|:----------------:|:---------------------:|
 | [fashion\_mnist\_tf\_keras](tutorials/fashion_mnist_tf_keras) | Fashion MNIST    | TensorFlow (tf.keras) |
 
+## Blog posts
+| Example                                 | Description  |
+|:---------------------------------------:|:------------:|
```
Comment (MikhailKardash) [inline README.md]: Should extend the `Description` field with more whitespace and more `-` to follow the formatting of the rest of the tables.

File: blog/python_sdk_demo/mednist_model/config.yaml (hunk 1 above)
Comment (wes-turner) [reply blog/python_sdk_demo/mednist_model/config.yaml]: done!

File: README.md (hunk 2 above)
Comment (wes-turner) [reply README.md]: Done

**Pull request merged**